src/bindings/python/src/openvino/__init__.py

```diff
@@ -3,3 +3,47 @@
 # SPDX-License-Identifier: Apache-2.0
 
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore # mypy issue #1422
+
```

Comment:
do we have other setup.py scripts which installs something in `openvino` namespace package, which actually can work without OpenVINO Python API?

Maybe accuracy checker? If yes, we cannot keep a single `openvino/__init__.py` here

Reply:
This is a good question. I also modified some packages on OMZ side (for now locally) and it works around the problem for `mo`. Need to double-check the rest.

Generally you refer to packaging styles. If we consider https://peps.python.org/pep-0420/#packaging-implications there are two options:

1) `__init__` files are involving same contents and it works as overriding them is always resulting in the same file.
2) There are no `__init__` files at all on namespace level.

Second option is what is implemented here. The promise is that everything will reside in same namespace `openvino` with only main repository being holding `__init__.py` file. This is currently working with `mo` for example.

I need to test it, this is why I put up this PR to have some runs on CIs and catch all issues.
BTW, this is the cost for requested change of namespace design. At least I do not know if there is another way around it.

Reply:
>>do we have other setup.py scripts which install something in openvino namespace package, which actually can work without OpenVINO Python API?
Maybe accuracy checker? 

yes, please take into account that accuracy checker can work without openvino installation and installed in openvino.tools